compiler/luci/pass/include/luci/CircleOptimizer.h

```diff
@@ -47,6 +47,7 @@ class CircleOptimizer final
       FuseMulWithConv,
       FuseMulWithDiv,
       FuseTransposeWithMean,
+      ResolveBuiltinOpAdd,
```

Comment:
The overall description of the new pass is as follows.

- The discussion of the New pass, `ResovleBuiltinOpAddPass`, came from here.
-> https://github.com/Samsung/ONE/issues/10784#issuecomment-1797862821
- This new pass is a pass that processes BroadcastTo(Builtin op) - Custom(AddV2) pattern.

Reply:
I can assume `ResolveCustomOpAdd` as resolve `CustomOp(Add)` to something processable Op/Ops.
I cannot catch what the meaning of `ResolveBuiltinOpAdd`.
- I cannot imagine what built-in op `Add` may have...

Reply:
@seanshpark 

> I can assume ResolveCustomOpAdd as resolve CustomOp(Add) to something processable Op/Ops.
I cannot catch what the meaning of ResolveBuiltinOpAdd.

I understand your opinion. :)
At first, I thought about it because I didn't know which pass name would be better. 
Similar to the existing `ResolveCustomAddPass`, the new pass name was considered
as `Resolve` + `Bulitin(BroadcastTo)` + `AddPass`.
but, I realized that the pass name is strange. 

so, I thought about another pass name candidate, how about this one?
- `ResolveBroadcastToCustomAddPass`
- `ResolveCustomAddPassV2`  ---> `V2` means `version`.
- ...?

Reply:
After some discussion with @seanshpark, we might as well implement the logic of this PR at the existing `ResolveCustomAddPass`.